tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import itertools
+from functools import partial
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tflite_layer_test_class import TFLiteLayerTest
+
+np.random.seed(42)
+
+
+def make_positive_array(inputs_dict):
+    for input in inputs_dict.keys():
+        inputs_dict[input] = np.random.randint(1, 10, inputs_dict[input]).astype(np.float32)
+    return inputs_dict
+
+
+def make_boolean_array(inputs_dict):
+    for input in inputs_dict.keys():
+        inputs_dict[input] = np.random.randint(0, 1, inputs_dict[input]) > 1
+    return inputs_dict
+
+
+data_generators = {
+    'positive': make_positive_array,
+    'boolean': make_boolean_array,
+}
+
+test_ops = [
+    {'op_name': 'ABS', 'op_func': tf.math.abs},
+    {'op_name': 'CAST', 'op_func': partial(tf.cast, dtype=tf.int32)},
+    {'op_name': 'CEIL', 'op_func': tf.math.ceil},
+    {'op_name': 'COS', 'op_func': tf.math.cos},
+    {'op_name': 'ELU', 'op_func': tf.nn.elu},
+    {'op_name': 'EXP', 'op_func': tf.math.exp, 'kwargs_to_prepare_input': 'positive'},
+    {'op_name': 'FLOOR', 'op_func': tf.math.floor},
+    {'op_name': 'LEAKY_RELU', 'op_func': partial(tf.nn.leaky_relu, alpha=-0.5)},
+    {'op_name': 'LOG', 'op_func': tf.math.log, 'kwargs_to_prepare_input': 'positive'},
+    {'op_name': 'LOG_SOFTMAX', 'op_func': partial(tf.nn.log_softmax, axis=-1)},
+    {'op_name': 'LOGICAL_NOT', 'op_func': tf.math.logical_not, 'kwargs_to_prepare_input': 'boolean', 'dtype': tf.bool},
+    {'op_name': 'LOGISTIC', 'op_func': tf.math.sigmoid},
+    {'op_name': 'NEG', 'op_func': tf.math.negative},
+    {'op_name': 'RELU6', 'op_func': tf.nn.relu6},
+    {'op_name': 'ROUND', 'op_func': tf.math.round},
+    {'op_name': 'RSQRT', 'op_func': tf.math.rsqrt, 'kwargs_to_prepare_input': 'positive'},
+    {'op_name': 'SIN', 'op_func': tf.math.sin},
+    {'op_name': 'SOFTMAX', 'op_func': partial(tf.nn.softmax, axis=-1)},  # additionally test with alpha
+    {'op_name': 'SQRT', 'op_func': tf.math.sqrt, 'kwargs_to_prepare_input': 'positive'},
+    {'op_name': 'SQUARE', 'op_func': tf.math.square},
+    {'op_name': 'TANH', 'op_func': tf.math.tanh},
+
+    # These operations are getting optimized out by tflite aka empty tfl model
+    # {'op_name': 'RANK', 'op_func': tf.rank},
+    # {'op_name': 'SHAPE', 'op_func': partial(tf.shape, out_type=tf.int32)},
+
+    # This op could not be converted standalone -- tries to become FlexOp (offload from tfl to tf)
+    # {'op_name': 'SIGN', 'op_func': tf.math.sign},
+
+    # TF has no such standalone operation
+    # {'op_name': 'HARD_SWISH'}
```

Comment:
Can we have a ticket for these JIRA cases - just don't forget about them?

Reply:
We have >100 operations to test. We will not forget about them -- we have a table, not JIRAs.